src/provider.ts

```diff
@@ -63,34 +63,40 @@ export class ProviderNotInitializedException extends Error {
 }
 
 export class ProviderRpcClient {
-    private readonly _properties: ProviderProperties;
+    private _provider?: SurfKeeperProvider;
 
-    private readonly _api: RawProviderApiMethods;
+    private readonly _properties: ProviderProperties;
 
     private readonly _subscribe: RawProviderSubscriptionMethods;
 
     private readonly _initializationPromise: Promise<void>;
 
-    private _provider?: Provider;
+    public readonly networks: RawProviderApiNetworks;
 
     constructor(properties: ProviderProperties = {}) {
         this._properties = properties;
 
         // Wrap provider requests
-        this._api = new Proxy<RawProviderApiMethods>({} as RawProviderApiMethods, {
-            get:
-                <M extends ApiMethod>(_object: RawProviderApiMethods, method: M) =>
-                (params: RawProviderApiRequestParams<M>) => {
-                    if (this._provider != null) {
-                        return this._provider.request({ method, params } as {
-                            method: RequestMethod;
-                            params: RequestParams<RequestMethod>;
-                        });
-                    } else {
-                        throw new ProviderNotInitializedException();
-                    }
-                },
-        });
+        this.networks = ['everscale', 'gosh', 'ton', 'venom', 'dev', 'fld'].reduce(
+            (api, networkKey) => ({
+                ...api,
+                [networkKey as ProviderNetwork]: new Proxy<RawProviderApiMethods>({} as RawProviderApiMethods, {
+                    get:
+                        <M extends ApiMethod>(_object: RawProviderApiMethods, method: M) =>
+                        (params: RawProviderApiRequestParams<M>) => {
+                            if (this._provider != null && this._provider[networkKey as ProviderNetwork]) {
+                                return this._provider[networkKey as ProviderNetwork].request({ method, params } as {
+                                    method: RequestMethod;
+                                    params: RequestParams<RequestMethod>;
```

Comment:
Should we send the "net" in params here? (since we know it)

Reply:
If I'm not mistaking there is no need to pass network here